src/zenml/integrations/kubernetes/orchestrators/kubernetes_orchestrator.py

```diff
@@ -337,16 +337,17 @@ def prepare_or_run_pipeline(
         Raises:
             RuntimeError: If trying to run from a Jupyter notebook.
         """
+        # TODO: consider making this a warning instead
         # First check whether the code is running in a notebook.
-        if Environment.in_notebook():
-            raise RuntimeError(
-                "The Kubernetes orchestrator cannot run pipelines in a notebook "
-                "environment. The reason is that it is non-trivial to create "
-                "a Docker image of a notebook. Please consider refactoring "
-                "your notebook cells into separate scripts in a Python module "
-                "and run the code outside of a notebook when using this "
-                "orchestrator."
-            )
+        # if Environment.in_notebook():
```

Comment:
We can just remove this right, no need to keep it commented out?

Reply:
yeah delete this and this is good to go imo